src/Sylius/Component/Review/Calculator/AverageRatingCalculatorInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Review\Calculator;
+
+use Sylius\Component\Review\Model\ReviewableInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface AverageRatingCalculatorInterface
```

Comment:
The same interface can be also used for `MedianCalculator`, `AverageRatingCalculator` etc. IMHO we should name it `RatingCalculatorInterface` or `ReviewableRatingCalculatorInterface`, wdyt?